File: server/Model/Audio.ts
import { Entity, PrimaryGeneratedColumn, Column, ManyToMany, OneToMany } from "typeorm";
import { Style } from "./Style";
import { Edit } from "./Edit";

@Entity()
export class Audio {
    @PrimaryGeneratedColumn()
    audio_id: number

    @Column()
    audio_credit: string

    @Column()
    audio_link: string

    @ManyToMany(() => Style, style => style.style_audios)
    audio_styles: Style[]

    @OneToMany(() => Edit, edit => edit.edit_audio)
    audio_edits: Edit[]

    @Column()
    audio_duration: number
}

File: server/src/main/java/laurriana/time2day/model/Audio.java
package laurriana.time2day.model;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.ArrayList;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Audio {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int audioId;

    private int audioDuration;

    private String audioName;

    private String audioCredit;

    private String audioUrl;

    @ManyToMany
    private ArrayList<Style> audioStyles;

    @OneToMany
    private ArrayList<Edit> audioEdits;
}
